src/leo-client-app/src/components/EditSchedules.tsx

```diff
@@ -1,32 +1,25 @@
 import { useRouter } from "next/router";
-import { sendCommandSchedule } from "@/constants/api";
+import { BACKEND_URL, sendCommandSchedule } from "@/constants/api";
```

Comment:
if you can do me a favour and rebase the pr i just did to import the backendurl like this instead of me manually defining it in [noradId].tsx and satellitesofinterest.tsx thatd be huge -> i imported it wrong

Reply:
yeah i rebased and did that